docs/react/use-custom-date-library.zh-CN.md

```diff
@@ -117,11 +117,63 @@ module.exports = {
 编写如下代码:
 
 ```tsx
-import dateFnsGenerateConfig from 'rc-picker/es/generate/dateFns';
 import generatePicker from 'antd/es/date-picker/generatePicker';
 import 'antd/es/date-picker/style/index';
+import dateFnsGenerateConfig from 'rc-picker/es/generate/dateFns';
 
 const DatePicker = generatePicker<Date>(dateFnsGenerateConfig);
 
 export default DatePicker;
 ```
+
+## 使用 luxon
+
+可以使用 [luxon](https://moment.github.io/luxon/) 代替 dayjs 并支持相同的功能，但它确实引入了一些我们将在下面解释的行为差异。
+
+### 执行
+
+创建一个 `src/components/DatePicker.tsx` 文件并实现基于 luxon 的选择器，如下所示：
+
+```tsx
+import generatePicker from 'antd/es/date-picker/generatePicker';
+import 'antd/es/date-picker/style/index';
+import type { DateTime } from 'luxon';
+import luxonGenerateConfig from 'rc-picker/lib/generate/luxon';
+
+const DatePicker = generatePicker<DateTime>(luxonGenerateConfig);
+
+export default DatePicker;
+```
+
+### 与 day-js 的显着差异
```

Comment:
```suggestion
### 与 day.js 的差异
```

Reply:
Thank you 🙏 Adjusted in this commit: b9e00ebb8f800723c3da2e02e40b1adace1925c5